gettsim/transfers/benefit_checks/vermoegens_checks.py

```diff
@@ -216,8 +242,8 @@ def freibetrag_vermögen_hh(
     """
     out = (
         freibetrag_vermögen_anspruch_hh
-        + anz_minderj_hh * arbeitsl_geld_2_params["vermögensfreibetrag"]["kind"]
+        + anz_minderj_hh * arbeitsl_geld_2_params["vermögensfreibetrag_kind"]
         + (haushaltsgröße_hh - anz_minderj_hh)
-        * arbeitsl_geld_2_params["vermögensfreibetrag"]["ausstattung"]
+        * arbeitsl_geld_2_params["vermögensfreibetrag_austattung"]
```

Comment:
The way it is used here, I am wondering why `vermögensfreibetrag_austattung` is not named `vermögensfreibetrag_erwachsene`. For Ausstattung I wouldn't have expected it to be the same for each adult.

We need to improve the description of this parameter (and maybe change the name?`).

Reply:
I looked it up and I think it needs to be added for the children as well:
"ein Freibetrag für notwendige Anschaffungen in Höhe von 750 Euro für jeden in der Bedarfsgemeinschaft lebenden Leistungsberechtigten." What do you think? Change it?

Reply:
I also understand it that way. We should change it then.

Reply:
Changed!